pkg/cmd/subcommand/install.go

```diff
@@ -161,17 +167,16 @@ func (i *Install) ValidateArgs(cmd *cobra.Command, args []string) error {
 
 func (i *Install) RunInstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 	operator := common.NewOperator(runtime.GOOS, i.OpenFunctionVersion, i.Timeout, i.RegionCN, i.Verbose)
-	ti := util.NewTaskInformer("")
 	continueFunc := func() bool {
 		reader := bufio.NewReader(os.Stdin)
-		fmt.Fprintln(w, ti.BeforeTask("You have used the `--upgrade` parameter, which means that the installation process "+
-			"will overwrite the components that already exist.\n"+
-			"Please ensure that you understand the meaning of this command "+
-			"and follow the prompts below to confirm the action.\n"+
-			"Enter 'y' to continue and 'n' to abort:"))
+		util.BeforeTask("You have used the `--upgrade` parameter, which means that the installation process " +
```

Comment:
"You have specified the `--upgrade` flag, which means that the installation process " +
			"will upgrade components currently installed.\n" +
			"Please make sure that you're aware of the consequences of this command " +
			"and follow the prompts below to confirm the upgrade.\n" +
			"Enter 'y' to continue and 'n' to abort:"